edgelet/contrib/centos/aziot-edge.spec

```diff
@@ -17,7 +17,7 @@ URL:            https://github.com/azure/iotedge
 %{?systemd_requires}
 BuildRequires:  systemd
 Requires(pre):  shadow-utils
-Requires:       aziot-identity-service >= 1.2.0~rc4-1
+Requires:       aziot-identity-service >= 1.2.0-1
```

Comment:
@arsing Was there some discussion about making these dependencies `=` instead of `>=`?

Reply:
If IS is to be a common component on which others beyond IoT Edge can depend then it must support a versioned API (it currently does) so that existing clients can continue to work with newer versions of IS.  If IoT Edge and some other component were to both demand a specific version and it isn't the same, then it puts a customer in the position of having to choose one or the other.  Recommend not making this `=` and keeping the `>=`.

Reply:
Yes, we want it to be `=` for now. Because of the tighter-than-originally-planned integration between the packages due to the `aziotctl` and `iotedge` CLIs, let's be conservative.

Reply:
@arsing this would be a temporary state and not a long-term position, correct?

Reply:
Yes, "for now".